packages/tempo/src/support/tempo.init.ts

```
@@ -0,0 +1,127 @@
+		const arg = asType(optVal);
+
+		switch (optKey) {
+			case 'snippet':
+			case 'layout':
+			case 'event':
+			case 'period':
+			case 'ignore':
+				if (!hasOwn(state.parse, optKey))
+					state.parse[optKey] = create(state.parse, optKey);
+
+				const rule = state.parse[optKey];
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)